### PR TITLE
Recognize Rack Symbols as Status Codes

### PR DESCRIPTION
**Pull request by alex-tan**
Addresses #466

### CODE REVIEW

**Comment by Tybot204** [comment]
:+1: Would love to see this merged in. I prefer Rack codes since I think it makes your code much clearer! I usually can't remember all of the codes and their meanings.

File: lib/apipie/error_description.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 module Apipie
 
   class ErrorDescription
-
     attr_reader :code, :description, :metadata
 
+    class UnrecognizedCode < Exception
```
Comment (iNecas) [inline lib/apipie/error_description.rb]: We have all other errors at one place her https://github.com/Apipie/apipie-rails/blob/296c097f9acb951149d3a67db69b84eb216bdd7e/lib/apipie/errors.rb. Also, prefer `StandardError`, or in this case even `Apipie::Error` over `Exception`

**Comment by iNecas** [comment]
Nice one. One comment inline, but should be good to go otherwise

**Comment by alex-tan** [comment]
@iNecas put the change into place! Thanks for reviewing.

**Comment by alex-tan** [comment]
Just an update: merged master back in and the build is green ✅

**Comment by iNecas** [comment]
Could you rebase you branch, rather than merging the master into your branch: it makes it simpler to test and we prefer rebase over merge in general. This branch has conflicts at the moment.

**Comment by alex-tan** [comment]
@iNecas done!

**Comment by iNecas** [comment]
Thanks @alex-tan

**Comment by alex-tan** [comment]
@iNecas 👍

**Comment by iNecas** [comment]
apipie-rails 0.5.1 released, including this change

**Pull request merged**